guides/user/canary/config/index.md

```diff
@@ -0,0 +1,169 @@
+ image_path="./enable_canary.png"
+%}
+
+## Create a canary configuration
+
+You can create as many of these as you like, and when you create a canary stage,
+you can select which configuration to use. Configurations you create within an
+application are available to all pipelines in that application, but your
+Spinnaker might be set up so that all configurations are available to all
+pipelines.
+
+1. Hover over the __Delivery__ tab, and select __Canary__.
+
+   ![Select __Canary__ from the __Delivery__ menu.](/guides/user/canary/config/delivery_menu_canary.png)
+1. Select __Add configuration__.
+
+1. Provide a __Name__ and __Description__.
+
+   This is the name shown in the stage config when you create a canary stage for your
+   pipeline.
+
+   ![Canary config declaration](/guides/user/canary/config/canary_config_create.png)
+
+The sections below describe how to specify the metrics and the scoring thresholds
+and weights.
+
+## Create metric groups and add metrics
+
+The metrics available depend on the telemetry provider you use. Spinnaker
+currently supports Stackdriver only.
+
+Metrics are evaluated even if they're not added to groups, but if you want to
+apply the weighting that determines the relative importance of different metrics,
+you need to [add them to groups](#create-metric-groups-and-add-metrics).
+
+1. In the __Metrics__ section, select __Add Metric__.
+
+1. Select the group to add this metric to.
+
+   Data is evaluated for all metrics, but metrics scores only affect the canary
+   evaluation for metrics that are grouped here.
+
+   Click __Add Group__ to create each group you'll use. For example, you might
+   create a group called  "cpu" and add a set of CPU-related metrics. Then,
+   when you configure a new metric here, you would select "cpu" for the
+   CPU-related metrics you add.
+
+1. Give the metric a name.
+
+1. Specify whether this metric fails when the value deviates too high or too low
+compared to the baseline.
+
+   Or select __either__, in which case it fails on deviation in either direction.
+
+1. Optionally, choose a [filter
+template](/guides/user/canary/config/filter_templates/).
+
+   This is only available  if your Spinnaker is [configured for it](). Filter
+   templates are collections of [Stackdriver monitoring
+   filters](https://cloud.google.com/monitoring/api/v3/filters).
+
+   Here's an example:
+
+   ```
+   resource.type = "gce_instance" AND
+   resource.labels.zone = starts_with("${zone}")
+   ```
+
+1. Identify the specific metric you're including in the analysis configuration:
+
+   * In the __Metric Type__ field type at least 3 characters to populate the
+   field with available metrics.
+
+     For example, if you type `cpu` you'll get a list of metrics avaialble from
+     your telemetry provider, from which you can choose.
+
+     ![List of available metrics](/guides/user/canary/config/metric_type_list_cpu.png)
+
+1. Optionally, click __Group by__ and enter the metric metadata attribute by
```

Comment:
This is specific to Prometheus and Stackdriver, so would consider splitting this out into provider metric configuration guides.

Reply:
@danielpeach 
[This](https://prometheus.io/docs/prometheus/latest/querying/basics/#instant-vector-selectors) is the best I could find for group-by in Prometheus. WDYT?